### ISSUE TITLE
关于train/dev/test的划分

### ISSUE THREAD

**Issue by carrie0307** [thunlp/Adv-ED#2]
作者您好，感谢您分享的代码。但是关于ACE2005数据集中train/dev/test的划分有些疑问，希望能得到您的解答：
 您在文中说到**we use the same test set containing 40 newswire
documents, development set with 30 randomly selected documents and training set with the remaining 529 documents**, 请问这40篇test具体是哪40篇呢?

根据以往文献，最早提出用40篇做test的是[Heng Ji and Ralph Grishman,2008](http://wing.comp.nus.edu.sg/~antho/P/P08/P08-1030.pdf),但似乎也没有具体说是哪40篇，请问这个是可以任意选取吗？

谢谢，希望得到您的答复！

祝好！

**Comment by Bakser** [comment]
您好，
我们选取的是发表时间最靠后的40篇。原因很容易理解，编写时间靠后的新闻有可能引用或参考之前的新闻，用他们做训练有data leakage的风险。所以应当把最后40篇作为测试集，前面的作为训练集。
谢谢您对我们工作的关注。

**Comment by carrie0307** [comment]
好的，谢谢您的回复~
